### ISSUE TITLE
[UX] "Available updates" page: Do not render the "Includes: ..." div (class="info") if it has only a single item.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1153]
This is especially obvious -not to mention completely unnecessary- in themes, where the "includes" div includes only the theme itself.

Compare before/after:

![backdrop-available_updates-info_div](https://cloud.githubusercontent.com/assets/2423362/9735279/fba2a728-567c-11e5-907b-2aac61f512cd.png)

![backdrop-available_updates-no_info_div_for_single_items](https://cloud.githubusercontent.com/assets/2423362/9735290/0fdab9ba-567d-11e5-8477-32e4af1837a2.png)

Less scrolling, less clutter, UX++

---
PR by @klonos: https://github.com/backdrop/backdrop/pull/1792

**Comment by klonos** [comment]
This was from back when I didn't know how to file PR's ...lets try to start closing those old issues 😄

**Comment by klonos** [comment]
@jenlampton assigning to you for review and to decide whether this is a task or feature request + then decide on a milestone. Thanx 😄

**Comment by jenlampton** [comment]
definitely a UX improvement, not really a feature. I would say it can go in as soon as it's ready :)

**Comment by klonos** [comment]
Tis ready now! One-line change (the rest is just comments added), all tests are green and it works! ...screenshot from the PR sandbox:

![backdrop-issue1153-pr1792](https://cloud.githubusercontent.com/assets/2423362/23046009/5d0213fa-f4fb-11e6-9878-7d104984308b.png)

Review and RTBC anyone please?

**Comment by klonos** [comment]
...or do you think perhaps reduce the bottom padding a bit so that the text is vertically centered? Could also make the "Includes" thing be a collapsible (and collapsed by default) fieldset.

Let me know.

**Comment by klonos** [comment]
Updated my PR. ...not a one-liner any more, but my previous code was only making sure that the text was not rendered if there is only one module included. It now prevents the entire wrapping `.includes` div from being rendered 😉 

...I also added checks for the rest of the divs (`.extras`, `.basethemes` and `.subthemes`) plus the `.info` div that wraps all these. So now there is no "padding" below the `.project` div:

![backdrop-issue1153-pr1792](https://cloud.githubusercontent.com/assets/2423362/23085929/a0b8c850-f5be-11e6-8303-f559a90c4966.png)
